app/src/main/java/com/bignerdranch/android/criminalintent/CrimeAdapter.kt

```diff
@@ -38,6 +50,7 @@ class CrimeAdapter(var crimes: List<Crime>) : RecyclerView.Adapter<CrimeAdapter.
       itemView.setOnClickListener {
         val text = context.getString(R.string.crime_title_pressed, crime.title)
```

Comment:
I think that the the toast should be handled by the fragment, what do you think?